src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+Item {
+    id: item
+
+    property var model: NC.SyncStatusModel {}
+
+    implicitHeight: layout.height
+
+    RowLayout {
+        id: layout
+
+        Layout.alignment: Qt.AlignLeft
+
+        width: item.width
+        spacing: 0
+
+        Image {
+            id: syncIcon
+
+            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+            Layout.topMargin: 16
+            Layout.bottomMargin: 16
+            Layout.leftMargin: 16
+
+            source: model.syncIcon
+            sourceSize.width: 32
+            sourceSize.height: 32
+        }
+
+        Item {
```

Comment:
this `Item` is not necessary and can be seen as bad as a poorly performant piece of `c++` code

Reply:
It is necessary to get the margins working correct. I tried hard without this `Item` but it never worked out well.

Reply:
Also, I remember that I had trouble centring the text without this item. Anyways, I will try it again.

Reply:
@mgallien I get it not working correctly without that `Item`.